main.tf

```diff
@@ -1,97 +1,92 @@
+
 # module "provider" {
 #   source = "./provider/scaleway"
 
 #   organization    = "${var.scaleway_organization}"
 #   token           = "${var.scaleway_token}"
-#   hosts           = "${var.hosts}"
+#   hosts           = "${var.node_count}"
 #   hostname_format = "${var.hostname_format}"
 #   region          = "${var.scaleway_region}"
 # }
 
 module "provider" {
   source = "./provider/hcloud"
 
-  hosts           = "${var.hosts}"
+  hosts           = "${var.node_count}"
   token           = "${var.hcloud_token}"
   type            = "${var.hcloud_type}"
   ssh_keys        = "${var.hcloud_ssh_keys}"
   location        = "${var.hcloud_location}"
   hostname_format = "${var.hostname_format}"
 }
 
-/*module "provider" {
-  source = "./provider/digitalocean"
+# module "provider" {
+#   source = "./provider/digitalocean"
 
-  token           = "${var.digitalocean_token}"
-  ssh_keys        = "${var.digitalocean_ssh_keys}"
-  hosts           = "${var.hosts}"
-  hostname_format = "${var.hostname_format}"
-  region          = "${var.digitalocean_region}"
-}*/
-
-module "dns" {
-  source = "./dns/cloudflare"
-
-  count      = "${var.hosts}"
-  email      = "${var.cloudflare_email}"
-  token      = "${var.cloudflare_token}"
-  domain     = "${var.domain}"
-  public_ips = "${module.provider.public_ips}"
-  hostnames  = "${module.provider.hostnames}"
-}
+#   token           = "${var.digitalocean_token}"
+#   ssh_keys        = "${var.digitalocean_ssh_keys}"
+#   hosts           = "${var.node_count}"
+#   hostname_format = "${var.hostname_format}"
+#   region          = "${var.digitalocean_region}"
+# }
 
-/*
-module "dns" {
-  source = "./dns/aws"
-
-  count      = "${var.hosts}"
-  access_key = "${var.aws_access_key}"
-  secret_key = "${var.aws_secret_key}"
-  region     = "${var.aws_region}"
-  domain     = "${var.domain}"
-  public_ips = "${module.provider.public_ips}"
-  hostnames  = "${module.provider.hostnames}"
-}
-*/
-
-/*
-module "dns" {
-  source = "./dns/google"
-
-  count        = "${var.hosts}"
-  project      = "${var.google_project}"
-  region       = "${var.google_region}"
-  creds_file   = "${var.google_credentials_file}"
-  managed_zone = "${var.google_managed_zone}"
-  domain       = "${var.domain}"
-  public_ips   = "${module.provider.public_ips}"
-  hostnames    = "${module.provider.hostnames}"
-}
-*/
-
-/*
-module "dns" {
-  source     = "./dns/digitalocean"
-  
-  count      = "${var.hosts}"
-  token      = "${var.digitalocean_token}"
-  domain     = "${var.domain}"
-  public_ips = "${module.provider.public_ips}"
-  hostnames  = "${module.provider.hostnames}"
-}
-*/
+# module "dns" {
+#   source = "./dns/cloudflare"
+
```

Comment:
If you wanna change style then comment-out empty lines belonging to blocks as well.

Reply:
Re-enable cloudflare, as this is required for accessing the cluster as described in the guide.